File: BeeCrowd/Python/beecrowd_1180.py
n = int(input())
data = list(map(int, input().split()))
for i in range(n):
	if (i == 0): 
	    menor = data[i];
	elif (data[i] < menor):
		menor, indicemenor = data[i], i

print("Menor valor: %d\nPosicao: %d" %(menor, indicemenor))

File: Cpp/beecrowd_1180.cpp
#include <stdio.h>

int main() {

  int menor, indicemenor, num;

  scanf("%d", &num);

  for (int x, i = 0; i < num; i++) {
    scanf("%d", &x);
    if (i == 0)
      menor = x;
    else if (x < menor) {
      menor = x;
      indicemenor = i;
    }
  }

  printf("Menor valor: %d\nPosicao: %d\n", menor, indicemenor);

  return 0;
}
